invenio_records_rest/serializers/datacite.py

```diff
@@ -26,29 +26,34 @@
 
 from __future__ import absolute_import, print_function
 
-from datacite import schema31
+from datacite import schema31, schema40
 from invenio_records.api import Record
 from lxml import etree
 from lxml.builder import E
 
 from .marshmallow import MarshmallowSerializer
 
 
-class DataCite31Serializer(MarshmallowSerializer):
+class BaseDataCiteSerializer(MarshmallowSerializer):
     """Marshmallow based DataCite serializer for records.
 
     Note: This serializer is not suitable for serializing large number of
     records.
     """
 
+    schema = None
+    """Class variable to define which schema use."""
+    version = 'Unknown'
```

Comment:
IMHO it would be more understandable to use `None` (or empty string). Basically something that evaluates to `False`.